.github/workflows/verify-build.yml

```diff
@@ -17,9 +17,13 @@ jobs:
         sudo apt -y install gcc-arm-none-eabi
     - name: Initialize submodules
       run: git submodule update --init --recursive
-#    - name: make simulator
-#      run: make simulator
-    - name: clean
-      run: make clean
     - name: make stm32f413
```

Comment:
Is there where we could change it so the default is "make stm32f413" but all we have to type is "make"?